nats-streaming-server/configuring/storelimits.md

```diff
@@ -138,3 +138,8 @@ On startup, the server will apply the store limits, which means that this channe
 
 We strongly recommend not raising the limit back to the higher limit if messages have been removed in the previous step because those removed messages may or may not become available again depending on the store implementation or if running in clustering mode or not.
 
+## Clustering
+When running --clustered, messages are kept in a RAFT consensus log file (under --cluster_log_path) in addition to the configured store.
```

Comment:
Use "\`" around `--clustered` and `--cluster_log_path` so they have different formatting.